e2e-polybft/e2e/apex_bridge_test.go

```diff
@@ -1617,3 +1691,58 @@ func TestE2E_ApexBridge_ValidScenarios_BigTests(t *testing.T) {
 		fmt.Printf("Prime - Success count: %v. prevAmount: %v. newAmount: %v. expectedAmount: %v\n", succeededCountPrime, prevAmountOnPrime, newAmountOnPrime, expectedAmountOnPrime)
 	})
 }
+
+func waitForIncludedInBatch(
```

Comment:
this looks like utility function (like `WaitForRequestStates`) move it where it belongs. Please check if something similar already exist, avoid duplicate code

Reply:
outdated